contracts/builder_unlock/src/contract.rs

```diff
@@ -423,7 +437,9 @@ mod helpers {
             Uint128::zero()
         }
         // Tokens unlock linearly between start time and end time
-        else if timestamp < schedule.start_time + schedule.duration {
+        else if (timestamp < schedule.start_time + schedule.cliff + schedule.duration)
```

Comment:
Let's move `+ schedule.cliff` to 436 line's right expression and the same to numerator of 443 line, to be consistent with the description of the Schedule structure. And then we can delete the if block starting from 458 line

Reply:
Fixed, thanks